### DISCUSSION TITLE
For m3 components not yet implemented, do we fall back to m2?

### DISCUSSION THREAD

**Discussion by EverybodyKurts** [material-components/material-web#5247] [General]
Hey all,

For components like card that are planned but not yet implemented for material web, should the m2 version of card be used? Any guidance is appreciated. Thank you :).

**Comment by CornerSyrup** [comment]
It's not auto fallback, but you can use MWC 2 alongside with 3, here's an example.

https://stackblitz.com/edit/web-platform-qrd1jz?file=index.html
